app/src/lib/app-state.ts

```diff
@@ -346,6 +346,10 @@ export enum RepositorySectionTab {
   History,
 }
 
+export interface IConflictState {
```

Comment:
This interface only exists to be used within another interface. I contemplated not exporting it just in case we wanted to do things like `let a: IConflictState | null = null`. Perhaps it's best not to assume we will need this and instead wait for the need to export this. I'll go ahead and add documentation, but it will be redundant as this is documented in the parent interface.